datastore/datastore.go

```diff
@@ -0,0 +1,260 @@
+package datastore
```

Comment:
Suggestion: rename to datastore-shim so that it communicates it is not a datastore, but something that sits between.

Reply:
Point taken, it’s not really a shim though. Proxy?

It does return a datastore though. What if I rename the file to proxy.go but leave it in the datastore package?

Reply:
Sounds good!